README.md

```diff
@@ -7,6 +7,21 @@ IRB stands for "interactive Ruby" and is a tool to interactively execute Ruby ex
 
 The `irb` command from your shell will start the interpreter.
 
+- [Installation](#installation)
+- [Usage](#usage)
+  - [The `irb` Executable](#the-irb-executable)
+  - [The `binding.irb` Breakpoint](#the-bindingirb-breakpoint)
+- [Commands](#commands)
```

Comment:
Looks good.
Adding navigation for `Debugging with IRB` section added in #689 will be better 😄

Reply:
Yeah I've rebased and updated it 👍